javax.naming.NameNotFoundException: subcontext
	at com.sun.jndi.fscontext.RefFSContext.getObjectFromBindings(RefFSContext.java:400)
	at com.sun.jndi.fscontext.RefFSContext.lookupObject(RefFSContext.java:327)
	at com.sun.jndi.fscontext.RefFSContext.lookup(RefFSContext.java:146)
	at com.sun.jndi.fscontext.FSContext.lookup(FSContext.java:127)
	at javax.naming.InitialContext.lookup(InitialContext.java:347)
	at com.topcoder.naming.jndiutility.JNDIUtils.getObject(JNDIUtils.java:403)
	at com.topcoder.naming.jndiutility.JNDIUtils.main(JNDIUtils.java:1125)
	at com.topcoder.naming.jndiutility.JNDIUtilsMainMethodTestCase.testMainWithTwoArguments(JNDIUtilsMainMethodTestCase.java:121)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:324)
	at junit.framework.TestCase.runTest(TestCase.java:164)
	at junit.framework.TestCase.runBare(TestCase.java:130)
	at junit.framework.TestResult$1.protect(TestResult.java:106)
	at junit.framework.TestResult.runProtected(TestResult.java:124)
	at junit.framework.TestResult.run(TestResult.java:109)
	at junit.framework.TestCase.run(TestCase.java:120)
	at junit.framework.TestSuite.runTest(TestSuite.java:230)
	at junit.framework.TestSuite.run(TestSuite.java:225)
	at junit.framework.TestSuite.runTest(TestSuite.java:230)
	at junit.framework.TestSuite.run(TestSuite.java:225)
	at junit.framework.TestSuite.runTest(TestSuite.java:230)
	at junit.framework.TestSuite.run(TestSuite.java:225)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:325)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:536)
